suite-native/graph/src/appGraphReducer.ts

```diff
@@ -0,0 +1,30 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
prefix it with app doesn't make sense, but anyway I would suggest just naming it reducer and moving it to package root

Reply:
Okay, changed it also for settings https://github.com/trezor/trezor-suite/pull/5838/commits/527b44a9dad6f51456a6b2cbcf93fb7fef034c36